origin-mobile/src/components/transaction-item.js

```diff
@@ -75,7 +75,7 @@ class TransactionItem extends Component {
       <TouchableHighlight onPress={handlePress}>
         <View style={[ styles.listItem, styles.completed, style ]}>
           {!picture && <View style={{ ...styles.thumbnail, ...styles.imageless }} />}
-          {picture && <Image source={{ uri: picture.url || picture }} style={styles.thumbnail} />}
+          {picture !== '' && <Image source={{ uri: picture.url || picture }} style={styles.thumbnail} />}
```

Comment:
@mikeshultz did you find a problem resulting from this loose checking for a `picture` value?

Reply:
For example, if `picture` is `undefined` or `null` for some reason, won't we end up with both the `View` and the `Image`?

Reply:
Yeah, this was hasty, but it was to tackle an actual bug.  When the user had no stored avatar and wasn't setting one, `picture` was an empty string.  And for whatever reason RN was trying to render that empty string causing errors.  I don't really have a full picture of what its values could be, however.

How about an explicit boolean conversion, instead?  e.g. `Boolean(picture) && [...]`